### PR TITLE
[CTD-314] added menu entries for Samba, RDP, and NextCloud.

### PR DESCRIPTION
**Pull request by Digital-Larry**

### CODE REVIEW

File: connectd/usr/bin/connectd_library [hunk 1]
```diff
@@ -635,13 +635,16 @@ protocolSelection()
     printf "     3) Secure Web (HTTPS) on port 443       \n"
     printf "     4) VNC on port 5900                     \n"
     printf "     5) nxWitness on port 7001               \n"
-    printf "     6) Custom (TCP)                         \n"
-    printf "     7) Return to previous menu              \n"
+    printf "     6) Samba (SMB) on port 445              \n"
+    printf "     7) Remote Desktop (RDP) on port 3389    \n"
+    printf "     8) NextCloud on port 80                 \n"
```
[the remark this answers is not in the document]
Comment (Digital-Larry) [reply connectd/usr/bin/connectd_library]: You mean nextcloud?  If so is it http or https?

[the remark this answers is not in the document]
Comment (danawoodman) [reply connectd/usr/bin/connectd_library]: @usk-tkhs can you please answer?

[the remark this answers is not in the document]
Comment (usk-tkhs) [reply connectd/usr/bin/connectd_library]: Default port of Nextcloud is 443 (HTTPS).

[the remark this answers is not in the document]
Comment (usk-tkhs) [reply connectd/usr/bin/connectd_library]: And, could you add OpenVPN too? Default port is 1194.

[the remark this answers is not in the document]
Comment (Digital-Larry) [reply connectd/usr/bin/connectd_library]: I changed NextCloud default port to 443.  Adding support for OpenVPN is a separate ticket and pull request.

**Pull request merged**